app/Jobs/TGForumGroups/SyncToForumGroup.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Jobs\TGForumGroups;
+
+use Alawrence\Ipboard\Exceptions\IpboardMemberIdInvalid;
+use Alawrence\Ipboard\Ipboard;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class SyncToForumGroup implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    protected $cid;
+    protected $forumGroup;
+
+    public function __construct(int $cid, int $forumGroup)
+    {
+        $this->cid = $cid;
+        $this->forumGroup = $forumGroup;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $ipboard = new Ipboard();
+
+        // Get the IP Board id based on the CID provided
+        // Need to think of a clever and clean way to get this...
```

Comment:
@atoff Any advice on this? I know you've done a fair amount of work on the Forum sync stuff in the past... I need to get the IPB member id based on the CID that is passed in?

Reply:
Would something like this work to load the Member object itself?

```
$member = \IPS\Member::load( $this->cid, 'p.field_12' );
```

Ref: https://invisioncommunity.com/4docs/advanced-usage/development/working-with-members-r179/